src/hw-monitor.h

```diff
@@ -271,49 +271,6 @@ namespace librealsense
     class hw_monitor
     {
     protected:
-        struct hwmon_cmd
-        {
-            uint8_t     cmd;
-            int         param1;
-            int         param2;
-            int         param3;
-            int         param4;
-            uint8_t     data[HW_MONITOR_BUFFER_SIZE];
-            int         sizeOfSendCommandData;
-            long        timeOut;
-            bool        require_response;
-            uint8_t     receivedCommandData[HW_MONITOR_BUFFER_SIZE];
-            size_t      receivedCommandDataLength;
-            uint8_t     receivedOpcode[4];
-
-            explicit hwmon_cmd(uint8_t cmd_id)
```

Comment:
Don't we use it?
Python maybe?

Reply:
Nope. No longer in use (I removed the only usage).